src/composables/addPost.js

```diff
@@ -0,0 +1,98 @@
+import { reactive, ref } from "vue";
+import slugify from "slugify";
+import { authenticate } from "@/utils/authenticate";
+import { useRegister } from "./register";
+import { usePostStore } from "@/stores/postStore";
+import router from "@/router";
+
+export const useAddPost = () => {
+  const { isErrorPresent } = useRegister();
+  const { clearData } = useRegister();
```

Comment:
const { isErrorPresent } = useRegister();
  const { clearData } = useRegister();
 
can't this import with a single initialization of useRegister? @aayushbatra02